### ISSUE TITLE
Codepen embeds only allow 6 character slugs

### ISSUE THREAD

**Issue by aspittel** [thepracticaldev/dev.to#734]
### Current Behavior

Codepens with only five characters in the URL aren't allowed.

### Expected Behavior

Codepens with only five characters in the URL aren't allowed. Old (?) Codepens have five characters in the url instead of six.

### Additional Info (device/browser, helpful links)

This line of code in [here](https://github.com/thepracticaldev/dev.to/blob/master/app/liquid_tags/codepen_tag.rb) just needs to be tweaked!

```js
/^(http|https):\/\/(codepen\.io)\/[a-zA-Z0-9\-]{1,20}\/pen\/([a-zA-Z]{6})\z/)&.zero?
```

[Here's](https://codepen.io/MarcoGuglielmelli/pen/eGCDn) a link to a CodePen with only 5 characters in the URL!

**Comment by benhalpern** [comment]
This has been fixed with #734

**Comment by aspittel** [comment]
Awesome, thanks!
